tsup.config.ts

```diff
@@ -17,7 +17,7 @@ export default defineConfig({
       "**/*.test.*",
     ],
   }),
-  format: ["cjs", "esm"],
+  format: "esm",
```

Comment:
Why did you use tsup instead of webpack? I chat-gpt'd it but curious what you say. Maybe it's obvious if you're building a components library you should use tsup.

Reply:
@williamnewman94 tsup is more straightforward than webpack: less configuration setup & features to setup. It's also recommended for libraries like ours